### ISSUE TITLE
Should we have a consistent plot theme?

### ISSUE THREAD

**Issue by malcolmbarrett** [r-causal/r-causal-blog#4]
In #3, I proposed some theming to the plots but we're not consistent across posts. Should we decide on a consistent theme/palette now? If I remember right, it's easier to declare even non-theme colors now, so we could just start each post with the same set of ggplot settings

**Comment by malcolmbarrett** [comment]
@LucyMcGowan it looks like you all have something automatic on the live free or dichotomize site?

**Comment by LucyMcGowan** [comment]
Yes! I wrote that a few months ago let me look back and remind myself what I did 😅
